website/docs/cloud-docs/api-docs/team-tokens.mdx

```diff
@@ -38,15 +38,271 @@ description: >-
+      "attributes": {
+        "created-at": "2024-06-19T18:44:44.051Z",
+        "last-used-at": null,
+        "description": "Team API token for team XYZ",
+        "token": null,
+        "expired-at": "2024-07-19T18:44:43.818Z"
+      },
+      "relationships": {
+        "team": {
+          "data": {
+            "id": "team-58pFiBffTLMxLphR",
+            "type": "teams"
+          }
+        },
+        "created-by": {
+          "data": {
+            "id": "user-ccU6h629hhzLJBpY",
+            "type": "users"
+          }
+        }
+      }
+    },
+  ]
+}
+```
+
+## Show a team token
+
+Use this endpoint to display a particular [team token](/terraform/cloud-docs/users-teams-organizations/teams#api-tokens).
+
+`GET /authentication-tokens/:token_id`
+
+| Parameter   | Description               |
+| ----------- | ------------------------- |
+| `:token_id` | The ID of the Team Token. |
+
+The object returned by this endpoint only contains metadata, and does not include the secret text of the authentication token. A token's secret test is only shown upon creation, and cannot be recovered later.
+
+| Status  | Response                                                | Reason                                                       |
+| ------- | ------------------------------------------------------- | ------------------------------------------------------------ |
+| [200][] | [JSON API document][] (`type: "authentication-tokens"`) | The request was successful                                   |
+| [404][] | [JSON API error object][]                               | Team Token not found, or unauthorized to view the Team Token |
+
+### Sample request
+
+```shell
+curl \
+  --header "Authorization: Bearer $TOKEN" \
+  --header "Content-Type: application/vnd.api+json" \
+  --request GET \
+  https://app.terraform.io/api/v2/authentication-tokens/at-6yEmxNAhaoQLH1Da
+```
+
+### Sample response
+
+```json
+{
+  "data": {
+    "id": "at-6yEmxNAhaoQLH1Da",
+    "type": "authentication-tokens",
+    "attributes": {
+      "created-at": "2017-11-29T19:18:09.976Z",
+      "last-used-at": null,
+      "description": "Team API token for team ABC",
+      "token": "QnbSxjjhVMHJgw.atlasv1.gxZnWIjI5j752DGqdwEUVLOFf0mtyaQ00H9bA1j90qWb254lEkQyOdfqqcq9zZL7Sm0",
+      "expired-at": "2023-04-06T12:00:00.000Z"
+    },
+    "relationships": {
+      "team": {
+        "data": {
+          "id": "team-LnREdjodkvZFGdXL",
+          "type": "teams"
+        }
+      },
+      "created-by": {
+        "data": {
+          "id": "user-MA4GL63FmYRpSFxa",
+          "type": "users"
+        }
+      }
+    }
+  }
+}
+```
+
+# Legacy team tokens API reference
 
 ## Generate a new team token
 
 Generates a new team token and overrides existing token if one exists.
 
 | Method | Path                                 |
 | :----- | :----------------------------------- |
-| POST   | /teams/:team_id/authentication-token |
+| POST   | /teams/:team_id/authentication-tokens |
```

Comment:
Is the legacy API endpoint changing? This would break existing scripts, wouldn't it?

Reply:
The legacy endpoints are not changing, and existing scripts using the legacy endpoint should not break! This looks like a typo and I've updated it - great catch 👀